packages/dashboard-backend/src/localRun/hooks/authorizationHooks.ts

```diff
@@ -15,7 +15,7 @@ import { FastifyInstance } from 'fastify';
 export function addAuthorizationHooks(server: FastifyInstance) {
   server.addHook('onResponse', (request, reply, done) => {
     if (
-      (request.url.startsWith('/api/') || request.url.startsWith('/dashboard/api/')) &&
```

Comment:
@olexii4 could you please clarify why do we need this change, as I recall `api` was added explicitly to prevent some redirection issue

Reply:
This code is only for a local run.
It removes the local environment variable CLUSTER_ACCESS_TOKEN when a response from '{CHE-Server}/api/' or '/dashboard/api/' has status code '401'.

 I added the following changes
```
(request.url.startsWith('/api/') || request.url.startsWith('/dashboard/api/')) &&
```
-> 
```
request.url.startsWith('/dashboard/api/') &&
```
because I already have a code that can return status code '401' from CHE-Server API:
```
 {CHE-Server}/api/oauth/token?oauth_provider=...
```